ql/src/java/org/apache/hadoop/hive/ql/exec/repl/util/ReplUtils.java

```diff
@@ -264,10 +230,6 @@ private static void addOpenTxnTaskForMigration(String actualDbName, String actua
           throws IOException, TException {
     List<Task<?>> taskList = new ArrayList<>();
     taskList.add(childTask);
-    if (isTableMigratingToTransactional(conf, tableObj) && updatedMetaDataTracker != null) {
```

Comment:
We can remove the entire method (addOpenTxnTaskForMigration) as it doesn't do anything apart from creating a list and adding child task to it. We can move that statement where this is getting called.